testing/integration/enroll_test.go

```diff
@@ -35,10 +37,16 @@ func TestEnrollAndLog(t *testing.T) {
 
 	t.Logf("In TestEnroll")
 	kibClient := info.KibanaClient
+
+	// Fleet API requires the namespace to be lowercased and not contain
+	// special characters.
+	policyNamespace := strings.ToLower(info.Namespace)
+	policyNamespace = regexp.MustCompile("[^a-zA-Z0-9]+").ReplaceAllString(policyNamespace, "")
+
```

Comment:
Thanks for fixing this.  There is identical code in `TestFQDN`.  Perhaps we should extract it into a helper function in `github.com/elastic/elastic-agent/pkg/testing/tools`?

Reply:
I think we should change the `info.Namespace` to always return a usable namespace. Removing the need for all other tests to have to worry about this.

Reply:
Yep, that's an even better idea.

Reply:
Yes I did a first fix and then when I got a failure on the TestFQDN I saw your code and bring it here.
I agree with @blakerouse suggestion, probably better to fix info.Namespace directly to avoid having to fix this in future test too.

Reply:
Done